### DISCUSSION TITLE
Bug programme cloudlink

### DISCUSSION THREAD

**Discussion by Eaielectronic** [community/community#125700] [Programming Help]
### Select Topic Area

Bug

### Body

Good morning, 
I installed python on a computer and I wanted to execute the following code by installing pip install cloudlink. 
# Import the server
from cloudlink import server

# Import protocols
from cloudlink.server.protocols import clpv4, scratch

# Instantiate the server object
server = server()

# Set logging level
server.logging.basicConfig(
    level=server.logging.INFO # See python's logging library for details on logging levels.
)

# Load protocols
clpv4 = clpv4(server)
scratch = scratch(server)

# Start the server!
server.run(ip="127.0.0.1", port=3000)
 The problem is that when I run the program, I get an error message that the line server=server().
And there is also an error message on the line from cloudlink.server.protocol import clpv4, scratch.
Thanks in advance

**Comment by t4lhawi** [comment]
Yeah

**Comment by Eaielectronic** [comment]
Le lien est le suivant vers le code [https://github.com/MikeDev101/cloudlink/wiki/Python-Server](url)

**Reply by Eaielectronic** [reply]
And I use my computer's port.
